utils/theme.js

```diff
@@ -13,7 +13,10 @@ const theme = {
     primaryMuted: '#86C6DE',
     primaryInactive: 'rgba(5, 184, 230, 0.5)',
     primaryMutedInactive: 'rgba(134, 198, 222, 0.5)',
-    selection: '#86C6DE'
+    selection: '#86C6DE',
+    wordClass: '#86C6DE',
```

Comment:
This is the same as `primaryMuted`. Why isn't `primaryMuted` used?

Reply:
I have no idea why "selection" is there, but I created a special colour for "wordClass" just to specify that that colour in particular was purely for the wordClass text which denotes noun (n) and verb (v)

Reply:
If we have too many separate colors I feel that we lose a bit of the point of having a theme. If both colors would change at the same time, to the same value, I think they should be using the same variable.

Selection is a separate variable because it should actually change at some point.

How do you feel? Both about how to use the theme, and if wordClass is distinctly different from primaryMuted.

Reply:
primaryMuted is being used for the title hence in my mind, I thought of them as two separate colours since I didn't think that if we changed the colour of wordClass, it would lead to a change in colour of the title for speak.exchange, but if both were to change at the same time then they should be the same primaryMuted colour for sure

Reply:
I agree that they do not necessarily change at the same time. That was a bad wording from me. I was thinking more in the sense of a complete theme change like dictionary.com and thesaurus.com, where you could use the same view components but just change themes.

For me it is more about cohesion, and that any time we want to change how wordClass looks we would have to change that in theme.js instead of where the view is. I would much more preffer that theme.js gives a complete set of any colors we would like to use, and if we want to change the color of wordClass, we do it in the view, by choosing another of the available colors. This way we get stronger cohesion, and easier git merges.

Reply:
You make a good point; I'll change it to primaryMuted